src/ethereum/mod.rs

```diff
@@ -105,6 +109,10 @@ pub struct Options {
     #[clap(long, env, default_value = "1000")]
     pub max_log_blocks: usize,
 
+    /// Minimum number of blocks before events are commited to db cache.
+    #[clap(long, env, default_value = "10")]
+    pub cache_blocks_delay: usize,
```

Comment:
This should apply to the whole module, not just caching. We currently do not do this for transactions, so I opened up #271 for that.

Reply:
I've renamed this option to `confirmation_blocks_delay` should be a bit more fitting name for #271. However, I think #271 should be out of scope of this PR. It will depend heavily on @kustosz work on #258.

Reply:
Agreed, it is out of scope for this PR.